src/components/notification/channel/discord.ts

```diff
@@ -25,10 +25,29 @@
 import axios from 'axios'
 
 import { WebhookData } from '../../../interfaces/data'
+import { NotificationMessage } from '../../../interfaces/notification'
 import { log } from '../../../utils/pino'
 
-export const sendDiscord = async (data: WebhookData): Promise<any> => {
+export const sendDiscord = async (
+  data: WebhookData,
+  message: NotificationMessage
+): Promise<any> => {
   try {
+    const notificationType =
+      message.meta.type[0].toUpperCase() + message.meta.type.substring(1)
+    let content
+    switch (message.meta.type) {
+      case 'incident':
+      case 'recovery': {
+        content = `New *\`${notificationType}\`* event from Monika\n\n${message.body}`
```

Comment:
Why do we use double newline here `\n\n` when we will trim it as one newline `\n` in the next line?

Reply:
updated @gsukanto